OPHD/Common.cpp

```diff
@@ -22,6 +22,21 @@
 using namespace NAS2D;
 
 
+std::string difficultyString(Difficulty difficulty)
+{
+	auto it = difficultyTable.value_comp();
```

Comment:
Was going to comment, this doesn't appear to be used anywhere, though it seems the Linux builds caught that already.

Also the name seems misleading as to what [`std::map::value_comp`](https://en.cppreference.com/w/cpp/container/map/value_comp) does. In particular, it returns a comparator, but the variable name would seem to imply an iterator. At any rate, I just learned about `value_comp`, which I didn't know existed before.

Reply:
Yeah, good call on the name. I was trying to use it as an iterator originally and didn't think about the name being off after I got it working. This function drove me to std::map instead of std::unordered_map since value_comp doesn't exist on unordered_map. I'll plan to fix the variable name in a different PR.